api/v1beta1/foundationdbcluster_types.go

```diff
@@ -2023,6 +2023,10 @@ type FoundationDBBackupSpec struct {
 	// This is measured in seconds. The default is 864,000, or 10 days.
 	SnapshotPeriodSeconds *int `json:"snapshotPeriodSeconds,omitempty"`
 
+	// DeploymentMetadata allows customizing labels and annotations on the
+	// deployment for the backup agents.
+	DeploymentMetadata *metav1.ObjectMeta `json:"deploymentMetadata,omitempty"`
```

Comment:
are we able to make this more generic to apply to other agents as well? Or at the very least can we rename it to something other than "deploymentMetadata" as it doesn't really make it clear that it's specific to the backup deployment.

Reply:
Yeah, that's fine. I can't think of any other deployments it would apply to either. I still think it should be renamed to backupMetadata or similar just so it's more clear